RELEASES.md

```diff
@@ -2,6 +2,27 @@
 
 > Note: ES3/IE8 compatibility will be removed in the future v3.x.x releases (scheduled for mid-late 2022), so if you need to retain ES3 compatibility you will need to remain on the 2.x.x versions of the SDK or your runtime will need install polyfill's to your ES3 environment before loading / initializing the SDK.
 
+## 3.3.7 (May 7th, 2025)
+
+### Changelog
+
+- #2518 Remove Generated docs from the repo
+- #2514 Address issues with isFeatureEnabled changes
+- #2517 Update Components to address governance issues
+- #2501 [Main][Task]31233527:Change Default RequestSizeLimitBytes
+- #2507 [main] Handle race condition during unload
+- #2513 [Main][Task]32698211: Add a Config to Allow Users to Change Max Number of Events Per Batch
+  - **Note**: New Config `maxEvtPerBatch` is added to the post channel `IChannelConfiguration`.
+- #2511 [Main][Task]31233527: Add a Config to Allow Users to Change RequestSizeLimitBytes
+  - **Note**: New Config `requestLimit` is added to the post channel `IChannelConfiguration`.
+- #2506 Drop correlation header to be passed on the dependency
+  - **Note**: Option to drop the enrichment of correlation header during dependency processing is added to `DependencyListenerFunction`
+- #2504 [main] [Click analytics] not logging no native html input elements
+  - **Note**: `trackElementTypes` is added to `IClickAnalyticsConfiguration` to allow additional, configurable HTML element types to be tracked in addition to the default set
+- #2451 [main] enable compress api in 1ds-post-channel and applicationinsights-channel
+  - **Note**: Using compress api feature is added through feature flag `zipPayload` and is currently disabled by default. See [more details](https://github.com/microsoft/ApplicationInsights-JS/blob/123ba4cd38f1478e91547d36c41668599834c734/shared/AppInsightsCore/src/JavaScriptSDK.Interfaces/IConfiguration.ts#L192)
```

Comment:
Suggestion: Add an extra note that this may be a breaking change **if** you are using a Proxy to redirect your telemetry to your own endpoint or are relying on the events to be uncompressed and that while this feature is initially disabled it is intended to be enabled by the service in the near future. So they should either update their collection endpoint to support GZip or they will need to explicitly disable the feature.

Include a link to the available features https://github.com/Microsoft/ApplicationInsights-JS?tab=readme-ov-file#feature